spec/_widget/components/article.spec.js

```diff
@@ -2,15 +2,16 @@ import { mount } from '@vue/test-utils';
 import Article from '../../../_widget/src/components/article/component.vue';
 
 describe('Article', () => {
-  const article = { id: '1', title: 'Title One', body: 'Article One' };
+  const article = { id: '1', title: 'Title One', body: 'Article One', sourceUrl: 'https://support.dnsimple.com' };
```

Comment:
It would be nice to move the specs from this file over to the `app.spec.js` as outside-in.